### DISCUSSION TITLE
DP-GEN trained model vs EAM Results

### DISCUSSION THREAD

**Discussion by vsa1920** [deepmodeling/deepmd-kit#3026] [Q&A] [answered]
Hi, everyone!
Firstly, thanks to everyone in this community for being very helpful.
I have been trying to do some analyses using [this](https://www.aissquare.com/models/detail?pageType=models&name=Cu-dpgen-model) DP-GEN generated model for Cu. The paper :
Yuzhi Zhang, Haidi Wang, Weijie Chen, Jinzhe Zeng, Linfeng Zhang, Han Wang, Weinan E,
DP-GEN: A concurrent learning platform for the generation of reliable deep learning based potential energy models

- can be found [here](https://www.sciencedirect.com/science/article/pii/S001046552030045X).

I analyzed the melting point and the thermal expansion coefficient of Copper using this potential in LAMMPS. And, for both of them, it did slightly worse than EAM potential in LAMMPS. My understanding was that DeePMD potentials are, in general, more accurate, considering they are almost on par with ab initio methods.

1) Is this expected? From the paper, I saw very promising results for some other quantities calculated for Cu.
2) Should I bother retraining this potential to do better?

**Comment by wanghan-iapcm** [comment, marked as answer]
The accuracy of the dp cannot exceed the dft method it was trained against. It is fairly possible that a highly accurate EAM is more accurate than dft (at least in predicting some of the properties). 
The xc functional used to label the Cu data should be pbe. If you want a more accurate melting point prediction, it worth trying more accurate xc functionals.

**Reply by vsa1920** [reply]
Hi! Thanks a lot for your response. That makes a lot of sense.  I never expected one of the authors of the paper to reply to my question. 
I have read the DeepPot-SE paper and have a question on it. I don't understand the intuitive explanation provided on how to choose M1 and M2 for making the encoded feature matrix from the embedding matrix. Is there any resource that would provide a deeper explanation? Thank you so much again!

**Reply by wanghan-iapcm** [reply]
No clear explanation. 
Three linearly independent vectors spans the space. Intuitively we set M2 >= 3.
